mlir/utils/jenkins/Jenkinsfile

```diff
@@ -242,7 +249,7 @@ pipeline {
                                 checkout scm
                                 buildProject('libMLIRMIOpen',
                                              '-DBUILD_FAT_LIBMLIRMIOPEN=ON')
-                                cmake arguments: '--install . --component libMLIRMIOpen --prefix /opt/rocm',\
+                                cmake arguments: "--install . --component libMLIRMIOpen --prefix ${WORKSPACE}/libMLIRMIOpen",\
```

Comment:
I prefer `/usr/local` as prefix over a local directory. This way, you wouldn't have to make another manual installation and MIOpen should be able to find this library in the right location.

Reply:
/usr/local is owned by root.

Reply:
Alternatively I think I can install everything at ~/usr/local to avoid manual installation.

Reply:
Sounds good, I'm good as long as we don't need another manual installation.

Reply:
@jerryyin What do you mean by "manual installation"?
I think having MIOpen go look for the libmlirmiopen.a file from MLIR's build directory is fine, as is it copying it into where it would be if the dependency was installed normally.

Reply:
@krzysz00 I was referring to [this comment](https://github.com/ROCmSoftwarePlatform/llvm-project-mlir/pull/455#discussion_r764936400) as the manual installation as the practice there was to another two bash script to move the installed targets again. This is brittle and will break if we altered the cmake installation logic.